### PR TITLE
CE-1544 SEO Check only on relevant asset in the story body

### PR DESCRIPTION
**Pull request by vpratti**

### CODE REVIEW

File: plugins/wordcount/plugin.js [hunk 1]
```diff
@@ -309,22 +333,6 @@ CKEDITOR.plugins.add("wordcount", {
             counterElement(editorInstance).className = "cke_path_item";
         }
 
-        function debounce(func, wait, immediate) {
```
Comment (radavi) [inline plugins/wordcount/plugin.js]: why is this removed? we need this? this is to improve the latency on the story editor? cc: @RyanCopley

Comment (vpratti) [reply plugins/wordcount/plugin.js]: Good Catch Raghu. I might have missed to pull down the latest.

**Review by radavi — APPROVED** [review]
